### PR TITLE
feat(float-button): [float-button] add float-button

### PR DESCRIPTION
**Pull request by fanbingbing16**
# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/tiny-vue/blob/dev/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

增加悬浮按钮组件
Issue Number: #1346 

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

**Comment by kagol** [comment]
@fanbingbing16 网站预览任务失败了，你merge下dev分支最新代码应该就好了
image

**Comment by fanbingbing16** [comment]
> @fanbingbing16 is attempting to deploy a commit to the **OpenTiny** Team on [Vercel](https://vercel.com).
> 
> A member of the Team first needs to [authorize it](https://vercel.com/git/authorize?team=OpenTiny&type=github&job=%7B%22headInfo%22%3A%7B%22sha%22%3A%22234a0ba7368354e974049267e8fc3ce93ea156a5%22%7D%2C%22id%22%3A%22QmeyTMqgmfrqXyrgKhVQuG3m6RN6Lw1KoSSbxShioFCD7U%22%2C%22org%22%3A%22opentiny%22%2C%22prId%22%3A1394%2C%22repo%22%3A%22tiny-vue%22%7D).

已merge

**Comment by fanbingbing16** [comment]
> @fanbingbing16 网站预览任务失败了，你merge下dev分支最新代码应该就好了 image

已merge

File: packages/vue/src/float-button/src/token.ts [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  'no-circle': 'sm:min-w-[theme(spacing.18)] pl-3 pr-3',
+  'is-circle': 'sm:min-w-[theme(spacing.18)] sm:rounded-full sm:pl-2 sm:pr-2',
+  'button-icon': '-mt-0.5 sm:text-base fill-current',
+  'button-icon-default': 'text-color-icon-primary hover:text-color-icon-hover active:text-color-icon-active',
+  'button-icon-disabled': 'text-color-icon-disabled hover:cursor-not-allowed',
+  'loading-svg': 'animate-spin-2 mr-1 fill-current -left-0.5 -right-0.5 -top-0.5 -bottom-0.5',
+  'button-link':
+    'text-color-link hover:text-color-link-hover active:color-link-hover active:hover:text-color-link-hover sm:hover:text-color-link-hover',
+  'button-banner': ' w-[calc(100%-theme(spacing.8))] mx-4'
+}
```
Comment (kagol) [inline packages/vue/src/float-button/src/token.ts]: @fanbingbing16 
这个文件的作用是什么呢？

File: .all-contributorsrc [hunk 2]
```diff
@@ -547,6 +547,7 @@
         "code"
       ]
     },
+
```
Comment (kagol) [inline .all-contributorsrc:550]: 这个文件不需要修改哈

File: packages/vue/src/float-button/src/token.ts (hunk 1 above)
Comment (fanbingbing16) [reply packages/vue/src/float-button/src/token.ts]: 没有用的，我忘记删除了

Comment (kagol) [reply packages/vue/src/float-button/src/token.ts]: > 没有用的，我忘记删除了

可以删除下

Comment (fanbingbing16) [reply packages/vue/src/float-button/src/token.ts]: > > 没有用的，我忘记删除了
> 
> 可以删除下

删除了

**Review by kagol — APPROVED** [review]

**Comment by kagol** [comment]
@fanbingbing16 我先合入了，后续可以完善下 e2e 测试用例

**Comment by fanbingbing16** [comment]
> @fanbingbing16 我先合入了，后续可以完善下 e2e 测试用例

好的，我会找时间写的

**Pull request merged**